src/Microsoft.PowerShell.Commands.Management/commands/management/Service.cs

```diff
@@ -2482,6 +2648,25 @@ internal static extern
             DWORD dwDesiredAccess
             );
 
+        [DllImport(PinvokeDllNames.QueryServiceConfigDllName, CharSet = CharSet.Unicode, SetLastError = true)]
```

Comment:
@daxian-dbw Are we still following the PInvokeDLLNames convention? It no longer makes sense since we stripped out the full CLR code

Reply:
@daxian-dbw Could you please comment?

Reply:
I think there is still one benefit -- you can easily know what native dependencies powershell has.